generator/discovery/update_discovery_doc.sh

```diff
@@ -35,8 +35,21 @@ readonly COMPUTE_DISCOVERY_JSON_RELATIVE_PATH="generator/discovery/compute_publi
 io::log_h2 "Fetching discovery document from ${COMPUTE_DISCOVERY_DOCUMENT_URL}"
 curl "${COMPUTE_DISCOVERY_DOCUMENT_URL}" >"${PROJECT_ROOT}/${COMPUTE_DISCOVERY_JSON_RELATIVE_PATH}"
 
+REVISION=$(sed -rn 's/  \"revision\": \"([[:digit:]]+)\",/\1/p' "${PROJECT_ROOT}/${COMPUTE_DISCOVERY_JSON_RELATIVE_PATH}")
+readonly REVISION
+io::log_h2 "git checkout -B update_compute_discovery_circa_${REVISION}"
```

Comment:
The log messages are usually descriptive rather than commands.  If you just want the command then perhaps `io::run` would be more appropriate.